dts/bindings/base/base.yaml

```diff
@@ -17,78 +26,169 @@ properties:
   compatible:
     type: string-array
     required: true
-    description: compatible strings
+    description: |
+      This property is a list of strings that essentially define what
+      type of hardware or other resource this devicetree node
+      represents. Each device driver checks for specific compatible
+      property values to find the devicetree nodes that represent
+      resources that the driver should manage.
+
+      The recommended format is "vendor,device", The "vendor" part is
+      an abbreviated name of the vendor. The "device" is usually from
+      the datasheet.
+
+      The compatible property can have multiple values. Additional values are
+      useful when the device is a specific instance of a more general family, to
+      allow the system to match from most- to least-specific device drivers.
```

Comment:
Perhaps explicitly mention that the list goes from "most specific" to "least specific"?

Reply:
please check reworded version